src/components/App.tsx

```diff
@@ -1,12 +1,71 @@
+/* eslint-disable no-restricted-syntax */
 import React, { ReactElement } from 'react';
 import '../style/App.css';
-import BlockChain from './BlockChain';
+import {
+  BrowserRouter as Router,
+  Link,
+  // Link,
+  Route, Switch,
+  // useHistory,
+} from 'react-router-dom';
+import DisplayUser from './DisplayUser';
+
+// import Submit from './Submit';
+// import UserCreation from './UserCreation';
 
 function App() : ReactElement {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  const handleSubmit = (
+    event: { preventDefault: () => void; },
+  ) => {
+    console.log(`
+      Email: ${email}
+      Password: ${password}
+    `);
+    event.preventDefault();
+  };
   return (
-    <div>
-      <BlockChain />
+    <Router>
+      <p>Home</p>
+      <Switch >
+        <Route exact path="/">
+          <div>
+            <form
+              onSubmit={handleSubmit}
+              // action="/display"
+            >
+              <h1>Create Account</h1>
+              <label>
+                Email:
+                <input
+                  name="email"
+                  type="email"
+                  onChange={(e) => setEmail(e.target.value)}
+                  required />
+              </label>
+              <label>
+                Password:
+                <input
+                  name="password"
+                  type="password"
+                  onChange={(e) => setPassword(e.target.value)}
+                  required />
+              </label>
+            </form>
+            <Link to="/display">
+              <button type='submit'>
+                  Submit
+              </button>
+            </Link>
     </div>
+          </Route>
+          <Route path="/display">
+            <DisplayUser userName={`Email:${email} et ${password}`}/>
```

Comment:
this should be a different component where you can see not only one user but all the available users list